bukkit/src/main/java/de/maxhenkel/voicechat/config/Translations.java

```diff
@@ -27,14 +27,14 @@ public Translations(ConfigBuilder builder) {
         );
         voicechatNotCompatibleMessage = builder.stringEntry(
                 "voicechat_not_compatible_message",
-                "Your voice chat version is not compatible with the servers version.\\nPlease install version %s of %s.",
+                "Your voice chat client version is not compatible with the server-side version.\\nPlease install version %s of %s.",
                 "The message a player gets when joining a server with an incompatible voice chat version",
                 "The first parameter is the mod/plugin version and the second parameter is the mod/plugin name"
         );
         voicechatNeededForCommandMessage = builder.stringEntry(
                 "voicechat_needed_for_command_message",
                 "You need to have %s installed on your client to use this command",
-                "The message a player gets when trying to execute a command that requires voice chat",
+                "The message a player gets when trying to execute a command that requires the voice chat mod installed on the client side.",
```

Comment:
this is inconsistent. Use *client-side*

Reply:
The phrase "server-side" in this file is hyphenated because it is a compound adjective that describes the noun "translations" or "version" (depending on the context). In contrast, the "client side" at the end of this sentence is not a compound adjective, so there is no need for a hyphen.

Reply:
Ok, sounds plausible :D